README.md

```diff
@@ -1,3 +1,22 @@
-# Example for Neuron Library
+# Serial example
+
+## Examples of controlling uart on ROScubeI based on mraa api.
+### Package 'examples_mraa_uart_controller':
+This package will create a node named 'uart_mraa_controller', and this node has two functions:
+1. This node will subscribe to a topic 'message_to_uart' and will write the message to uart port. 
+2. This node will listen to the uart port every 500ms and will publish the message from the uart to the topic 'message_from_uart'.
+
```

Comment:
You should put the whole command here, or users don't know how to use your example.
Like, `ros2 run <package_name> <executable>`